wallet/documentation/docs/how_tos/outputs_and_transactions/06_create_alias_output.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Create an Alias Output
+description: 'How to create an alias output using wallet.rs'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- outputs
+- create
+- create alias output
+- nodejs
+- python
+- java
+- rust
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import NodejsCode from "../../libraries/nodejs/how_to/outputs_and_transactions/_06_create_alias/_code_sample.mdx";
+import NodejsOutput from "../../libraries/nodejs/how_to/outputs_and_transactions/_06_create_alias/_expected_output.md";
+import PythonCode from "../../libraries/python/how_to/outputs_and_transactions/_06_create_alias/_code_sample.mdx";
+import PythonOutput from "../../libraries/python/how_to/outputs_and_transactions/_06_create_alias/_expected_output.md";
+import JavaCode from "../../_admonitions/_guide_coming_soon.md";
+import JavaOutput from "../../_admonitions/_guide_coming_soon.md";
+import RustCode from "../../libraries/rust/how_to/outputs_and_transactions/_06_create_alias/_code_sample.mdx";
+import RustOutput from "../../libraries/rust/how_to/outputs_and_transactions/_06_create_alias/_expected_output.md";
+
+You can create an alias output using the `Account.create_alias(alias_id, options)` function.
```

Comment:
Such a function doesn't exist
There is `create_alias_output()`, but it also doesn't accept an alias_id

Reply:
```suggestion
You can create an alias output using the `Account.create_alias_output(alias_options, transaction_options)` function.
```